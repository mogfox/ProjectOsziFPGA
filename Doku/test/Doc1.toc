\contentsline {section}{\numberline {1}Einleitung}{1}{section.1}%
\contentsline {section}{\numberline {2}Entwicklung}{1}{section.2}%
\contentsline {subsection}{\numberline {2.1}erste Schrittte}{1}{subsection.2.1}%
\contentsline {subsubsection}{\numberline {2.1.1}Verwendung der VGA Schnittstelle}{1}{subsubsection.2.1.1}%
\contentsline {subsubsection}{\numberline {2.1.2}Verwendung der internen ADCs}{1}{subsubsection.2.1.2}%
\contentsline {paragraph}{ADC Hardware}{2}{section*.2}%
\contentsline {paragraph}{ADC VHDL-Implementierung}{3}{section*.3}%
\contentsline {section}{\numberline {3}Quellen und Hilfsmittel}{3}{section.3}%
