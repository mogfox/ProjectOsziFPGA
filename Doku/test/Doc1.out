\BOOKMARK [1][-]{section.1}{\376\377\000E\000i\000n\000l\000e\000i\000t\000u\000n\000g}{}% 1
\BOOKMARK [1][-]{section.2}{\376\377\000E\000n\000t\000w\000i\000c\000k\000l\000u\000n\000g}{}% 2
\BOOKMARK [2][-]{subsection.2.1}{\376\377\000e\000r\000s\000t\000e\000\040\000S\000c\000h\000r\000i\000t\000t\000t\000e}{section.2}% 3
\BOOKMARK [3][-]{subsubsection.2.1.1}{\376\377\000V\000e\000r\000w\000e\000n\000d\000u\000n\000g\000\040\000d\000e\000r\000\040\000V\000G\000A\000\040\000S\000c\000h\000n\000i\000t\000t\000s\000t\000e\000l\000l\000e}{subsection.2.1}% 4
\BOOKMARK [3][-]{subsubsection.2.1.2}{\376\377\000V\000e\000r\000w\000e\000n\000d\000u\000n\000g\000\040\000d\000e\000r\000\040\000i\000n\000t\000e\000r\000n\000e\000n\000\040\000A\000D\000C\000s}{subsection.2.1}% 5
\BOOKMARK [1][-]{section.3}{\376\377\000Q\000u\000e\000l\000l\000e\000n\000\040\000u\000n\000d\000\040\000H\000i\000l\000f\000s\000m\000i\000t\000t\000e\000l}{}% 6
